pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -59,16 +70,45 @@ func (c *podSecurityOperatorConditions) addViolation(ns *corev1.Namespace) {
 	c.violatingCustomerNamespaces = append(c.violatingCustomerNamespaces, ns.Name)
 }
 
-func makeCondition(conditionType string, namespaces []string) operatorv1.OperatorCondition {
+func (c *podSecurityOperatorConditions) addInconclusive(ns *corev1.Namespace) {
+	if runLevelZeroNamespaces.Has(ns.Name) {
+		c.inconclusiveRunLevelZeroNamespaces = append(c.inconclusiveRunLevelZeroNamespaces, ns.Name)
+		return
+	}
+
+	isOpenShift := strings.HasPrefix(ns.Name, "openshift")
+	if isOpenShift {
+		c.inconclusiveOpenShiftNamespaces = append(c.inconclusiveOpenShiftNamespaces, ns.Name)
+		return
+	}
+
+	if ns.Labels[labelSyncControlLabel] == "false" {
+		// This is the only case in which the controller wouldn't enforce the pod security standards.
+		c.inconclusiveDisabledSyncerNamespaces = append(c.inconclusiveDisabledSyncerNamespaces, ns.Name)
+		return
+	}
+
+	c.inconclusiveCustomerNamespaces = append(c.inconclusiveCustomerNamespaces, ns.Name)
+}
+
+func makeCondition(conditionType, conditionReason string, namespaces []string) operatorv1.OperatorCondition {
+	var messageFormatter string
+
+	if conditionReason == violationReason {
+		messageFormatter = "Violations detected in namespaces: %v"
+	} else if conditionReason == inconclusiveReason {
+		messageFormatter = "Could not evaluate violations for namespaces: %v"
```

Comment:
Hm, we might need to read the [CFE documentation](https://github.com/openshift/enhancements/blob/995b620cb04c030bf62c908e188472fe7031a704/dev-guide/cluster-fleet-evaluation.md) says about those messages. E.g. I remember that we need to be careful with those messages under certain circumstances.

Reply:
I might be missing something, but I'm not seeing a format for the condition messages defined somewhere... For violations, this should format exactly the same as it always has - only the inconclusive part is new.

Reply:
I think we might be mixing up the `Reason` and the `Message` fields.

Before this PR, we were using `PSViolationsDetected` for the condition's `Reason` and `Violations detected in namespaces: %v` for the condition's `Message` (for all types of violations).

Now, `Reason` can be `PSViolationsDetected` or `PSViolationDecisionInconclusive` and
`Message` can be a few different strings... Are you saying that we now need to break `Reason` out by violation type?